index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cash Register</title>
+    <style>
+        body {
+            background-color: rgb(0, 0, 41);
+            color: white;
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;
+        }
+
+        header {
+            text-align: center;
+            padding: 20px;
+        }
+
+        h1 {
+            font-size: 48px;
+            margin: 0;
+            color: rgb(248, 243, 243);
+        }
+
+        table {
+            border-collapse: collapse;
+            width: 100%;
+            margin-bottom: 20px;
+        }
+
+        th,
+        td {
+            padding: 10px;
+            text-align: left;
+        }
+
+        th {
+            color: white;
+        }
+
+        tr:nth-child(even) {
+            background-color: rgb(1, 1, 43);
+        }
+
+        #container {
+            background-color: rgb(4, 139, 250);
+            padding: 20px;
+            border-radius: 10px;
+            max-width: 800px;
+            margin: 0 auto;
+        }
+        
+        #total { 
+            color: white;
+            text-align: center;
+            font-size: 25px;
+        } 
+        #container-label {
+            margin-bottom: 20px;
+        }
+
+        label {
+            display: block;
+            margin-bottom: 10px;
+        }
+
+        input[type="number"] {
+            padding: 10px;
+            border: none;
+            border-radius: 5px;
+            width: 100%;
+        }
+
+        #caisse {
+            background-color: rgb(0, 93, 168);
+            padding: 20px;
+            border-radius: 10px;
+        }
+
+        .touch-row {
+            display: flex;
+            flex-wrap: wrap;
+            justify-content: space-between;
+        }
+
+        .key {
+            width: 30%;
+            margin-bottom: 10px;
+        }
+
+        #tiroirCaisse {
+            text-align: center;
+            margin-top: 20px;
+        }
+
+        #buttonBlack {
+            display: inline-block;
+            width: 50px;
+            height: 50px;
+            background-color: black;
+        }
+
+        @media only screen and (max-width: 600px) {
+            h1 {
+                font-size: 36px;
+            }
+
+            table {
+                font-size: 14px;
+            }
+
+            input[type="number"] {
+                width: 70%;
+            }
+
+            .touch-row {
+                flex-direction: column;
+            }
+
+            .key {
+                width: 100%;
+            }
+        }
+    </style>
```

Comment:
- I told you in previous comments to separate your styling code from your HTML file as you have a CSS file for them.
- Solve this issue and then directly merge your branch.